src/banners/_base.scss

```diff
@@ -22,6 +22,6 @@
   }
 }
 
-[data-banner="background"] {
+.banner--background {
```

Comment:
I'm not married to this class. It doesn't feel quite clear enough. Would rather let the team decide on this though.